docs/src/pages/components/pickers/MaterialUIPickers.tsx

```diff
@@ -1,40 +1,44 @@
 import 'date-fns';
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
-import { makeStyles, createStyles } from '@material-ui/core/styles';
 import DateFnsUtils from '@date-io/date-fns';
 import {
   MuiPickersUtilsProvider,
   KeyboardTimePicker,
   KeyboardDatePicker,
 } from '@material-ui/pickers';
 
-const useStyles = makeStyles(
-  createStyles({
-    grid: {
-      width: '60%',
-    },
-  }),
-);
-
 export default function MaterialUIPickers() {
   // The first commit of Material-UI
   const [selectedDate, setSelectedDate] = React.useState<Date | null>(
     new Date('2014-08-18T21:11:54'),
   );
-  const classes = useStyles();
 
   function handleDateChange(date: Date | null) {
     setSelectedDate(date);
   }
 
   return (
     <MuiPickersUtilsProvider utils={DateFnsUtils}>
-      <Grid container className={classes.grid} justify="space-around">
+      <Grid container justify="space-around">
+        <KeyboardDatePicker
+          disableToolbar
+          variant="inline"
+          format="MM/dd/yyyy"
```

Comment:
```suggestion
          format="yyyy-MM-dd"
```
Should probably use the ISO standard instead of that horror show.

https://en.wikipedia.org/wiki/ISO_8601
https://en.wikipedia.org/wiki/Date_format_by_country

Reply:
USA is m/d/yyyy, this seems correct?

Reply:
>USA is m/d/yyyy, this seems correct?

They do use that backwards format but there is an extremely high chance that they are developers and are used to the ISO standard.

> USA is our main % source of traffic

I think that it would be better to use the same standard as the language(JavaScript) and not what a specific place uses. However it's just a demo and doesn't really matter.

Reply:
The reasoning was the following. Google "datepicker" copy the format used by the top results. I haven't looked past that. I think that the ISO format is good for machines.